src/last.js

```diff
@@ -0,0 +1,5 @@
+const lastArray = (array) =>{
```

Comment:
Same as above. If you are remaking lodash `last()` please name your function `last()`.